File: 5/5.php
<?php
$masa =  readline("Introduce tu peso en Kg : ");
$altura = readline("Introduce tu altura en Mts: ");

if(is_numeric($masa) && is_numeric($altura)){
    $total = floatval($masa) / pow(floatval($altura),2);
    echo "Tu índice de masa corporal es ".round($total,2);
}else{
    echo "Los valores deben ser numéricos";
}

File: 5/5.py
try:
    masa =  int(input("Introduce tu peso en Kg : "))
    altura = float(input("Introduce tu altura en Mts: "))
    if(type(masa)==int and type(altura)==float):
            total = (masa) / pow(altura,2)
            print("Tu índice de masa corporal es "+ str(round(total,2)))
except ValueError:
    print("Los valores deben ser numéricos")